PWGJE/EMCALJetTasks/macros/JetQA/plotPWGJEQA.py

```diff
@@ -1651,17 +1667,43 @@ def plotPtHard(f, qaList, outputDir, fileFormat):
   outputFilename = os.path.join(outputDirPtHard, "hPtHard" + fileFormat)
   plotHist(hPtHard, outputFilename, "hist", True)
 
+  #if a reference is provided
+  if qaListRef:
+  
+    hPtHardRef = qaListRef.FindObject("hPtHard")
+  
+    outputFilename = os.path.join(outputDirPtHard, "hPtHard_Ratio" + fileFormat)
+    xRangeMax = 100
+    yAxisTitle = "#frac{1}{N_{evts}}#frac{dN}{dp_{T}} [GeV^{-1}]"
+    legendTitle = "pT Hard production"
+    legendRunLabel = "hPtHard this run"
+    legendRefLabel = "hPtHard all runs"
+    ratioYAxisTitle = "Ratio: run / all runs"
+    hPtHardRef.SetLineColor(1)
+
+    ispp=1
+    if nEventsRef!=0:
+      plotSpectra(hPtHard, hPtHardRef,0x0, 0x0, nEvents, nEventsRef, ispp, xRangeMax, yAxisTitle, legendTitle, legendRunLabel, legendRefLabel, ratioYAxisTitle, outputFilename, "1", "2", "3")
+
 ########################################################################################################
 # Plot basic histogram    ##############################################################################
 ########################################################################################################
-def plotHist(h, outputFilename, drawOptions = "", setLogy = False, setLogz = False):
+def plotHist(h, outputFilename, drawOptions = "", setLogy = False, setLogz = False, yMin=-1):
  
   c = ROOT.TCanvas("c","c: hist",600,450)
   c.cd()
   if setLogy:
     c.SetLogy()
   if setLogz:
     c.SetLogz()
+  maxBin=h.GetBinContent(h.GetMaximumBin())
+  if yMin!=-1:
```

Comment:
We should avoid coding the logic like this, since it introduces a pathology for yMin=-1...

Reply:
deleted this option since it is only used once